Source/SwiftLintFramework/Models/SwiftVersion.swift

```diff
@@ -23,4 +23,11 @@ enum SwiftVersion {
 
         fatalError("Unexpected Swift version")
     }()
+
+    var nameKey: String {
```

Comment:
I think `nameKey` isn't representative of the conditional value it returns. Copying the code inline avoids coming up with an accurate name for it. So either come up with a good name, or skirt around the issue :-)